### ISSUE TITLE
Window isn't FrameworkElement?

### ISSUE THREAD

**Issue by michael-hawker** [sonnemaf/WinUI3AttachedPropertyCrashing#1]
Hi @sonnemaf, stumbled upon this in my feed. My initial guess is this is because Window isn't a `FrameworkElement`? If you copy your code you expect to work into a `Page`, does it work?

Feel like this isn't the original issue I was looking for, but it's probably related: https://github.com/microsoft/microsoft-ui-xaml/issues/4966

**Comment by sonnemaf** [comment]
No. With the help of @AndrewKeepCoding I was able to solve it. The class has to be a **public** class. Mine was internal and then it crashes.
